scripts/extract_download_logs.py

```diff
@@ -57,7 +80,32 @@
     "rp_start",
     "rp_end",
 ]
-OUTPUT_FILENAME = options.filename
+OUTPUT_FILENAME = args.filename
+
+
+def send_notify(
+    from_date: datetime.datetime,
+    to_date: datetime.datetime,
+    file: StringIO,
+    api_key: str = os.environ["NOTIFY_API_KEY"],
+    template_id: str = "196e5553-886c-40bd-ac9a-981a7868301b",
+    email_address: str = os.getenv("email", "test@example.com"),
+):
+    from_date_formatted = from_date.replace(microsecond=0).isoformat()
+    to_date_formatted = to_date.replace(microsecond=0).isoformat()
+    notifications_client = NotificationsAPIClient(api_key)
+    notifications_client.send_email_notification(
```

Comment:
The Notify API client appears to return the JSON response of the request from `send_email_notification`, so I wonder if we could do something with this to verify that everything went okay? For example I was able to run the script through successfully, with it saying "File sent via Notify", when I didn't actually receive any email

Reply:
It gives back an ID but then that would need to be polled asynchronously and could be queued for quite a long time so not sure we'd want to block the script for this, see: https://docs.notifications.service.gov.uk/python.html#email-status-descriptions